### DISCUSSION TITLE
1.3.1 Auto tilt  current mesh

### DISCUSSION THREAD

**Discussion by Timbo200** [Jyers/Marlin#599] [General Discussion]
On 1.3.1 Auto tilt the current mesh now probes only three points.  On 1.3.0 it probed 4 points.

Is this the desired behavior? Also minor bug (or perhaps not) that the probing 3/3 points on the display never clears itself after doing other activity in the printer menus.

Naturally the new behavior had the 3rd probe point right on top of the clip for my glass bed.  Easily solved by shifting the clip over,  but it's going to mess with my OCD  ;-)

**Comment by gothcha** [comment]
Yes it is, you can change the number of tilting points in the menu under tilting mesh, change the value to 2 to use 4 points as before.
